securedrop_client/app.py

```diff
@@ -120,6 +124,7 @@ def start_app(args, qt_args) -> None:
     - configure the client (logic) object.
     - ensure the application is setup in the default safe starting state.
     """
+    print(args.proxy)
```

Comment:
Looks like you left a debug `print` here.

Reply:
Yup, will be removed.